### ISSUE TITLE
Include ability to retrieve all coiled-coil or disordered regions via advanced search.

### ISSUE THREAD

**Issue by ValWood** [pombase/website#1720]
Request from Alison Pidoux via helpdesk.

**Comment by kimrutherford** [comment]
Would it be OK to have a query type like "Number of TM domains" but for each of coiled-coils, disordered regions and low complexity regions?  "Number of disordered regions" doesn't make as much sense as "Number of TM domains" but it would make for a consistent interface.

**Comment by ValWood** [comment]
I guess that would be OK because some proteins do have multiple low complexity and disordered regions.
like https://www.pombase.org/gene/SPBC428.08c
and if people want any disordered we can prompt with the range
(like max TMM is 18 I think, or somewhere around there)

**Comment by ValWood** [comment]
Removed discuss,  if you still have questions let me know will re discuss on next call.

**Comment by kimrutherford** [comment]
This is working for me on my desktop.  I've been tweaking the style of the page a bit too.

![query-buiilder-sub-nodes-2](https://user-images.githubusercontent.com/90474/134756499-137c8802-298a-4aca-81c9-3c7a535b61e3.png)

![query-buiilder-sub-nodes-1](https://user-images.githubusercontent.com/90474/134756522-c32d79fc-84a1-4501-8a2a-47063864d82c.png)

I've been amusing myself by looking at pombe genes with lots of coiled-coils, disordered regions or low complexity regions and then having a look in AlphaFold.  Like these:

https://alphafold.ebi.ac.uk/entry/Q9URU4

![alphafold-1](https://user-images.githubusercontent.com/90474/134756861-cd9ed07e-33e1-4eba-a898-ce91ae6dd655.png)

https://alphafold.ebi.ac.uk/entry/O74424

![alphafold-2](https://user-images.githubusercontent.com/90474/134756866-bc0ca1e6-a2ea-4f0b-af02-fe9cd7956a7f.png)

**Comment by kimrutherford** [comment]
My new favourite.  I'm not sure I trust this predicted structure.

(Sorry, I'll stop posting these now)

https://alphafold.ebi.ac.uk/entry/Q6LA55   SPAC3H5.09c

![alphafold-3](https://user-images.githubusercontent.com/90474/134757430-d0fb71c8-bd6e-4e65-9dcc-c53338295b8b.png)

**Comment by ValWood** [comment]
SPAC3H5.09c does look like a bit of a traffic jam, but the rest with packed beta sheets almost end to end is very believable. @snezhkaoliferenko  this is 
ER-PM contact site phospholipid-binding protein Hob2 
Any thoughts if this is in any way accurate?

**Comment by ValWood** [comment]
https://alphafold.ebi.ac.uk/entry/O74424 is  a nucleoporin (nup211), nuclear basket filament, so this isn't crazy either:
<img width="1022" alt="Screenshot 2021-09-25 at 10 20 15" src="https://user-images.githubusercontent.com/7359272/134766556-44c6d6c5-7570-402c-8e2a-9342b4922693.png">

**Comment by ValWood** [comment]
I like https://alphafold.ebi.ac.uk/entry/Q9URU4 pfl4  best, this provides a mechanism for recognition as the modification sites will be in the same place on each repeat (like velcro)

**Comment by ValWood** [comment]
Aw snap
https://alphafold.ebi.ac.uk/entry/Q96WV6
(alpha fold flummoxed by crazy protein)

**Comment by ValWood** [comment]
Back to the question, I like the new layout!

**Comment by kimrutherford** [comment]
> alpha fold flummoxed by crazy protein

I was disappointed by that one too.

I think they should have a page for each protein they've failed to predict where they own up.  :-)

**Comment by snezhkaoliferenko** [comment]
bit crazy can't figure out how the TM is supposed to be situated look at the AH in the top right corner
<img width="856" alt="image" src="https://user-images.githubusercontent.com/87086671/134771186-a821b97f-c3f7-4c37-9ce8-e8f026b7c80b.png">

**Comment by kimrutherford** [comment]
> Back to the question, I like the new layout!

If you'd like to give it a go, it's available from my desktop from 8 or 9pm until noon-ish each day:
https://desktop.kmr.nz/query

**Comment by ValWood** [comment]
Apparantly AlphFold is not good at taking TMMs into account in the models. It's an acknowledge problem.
Hopefully they will model this  in future itereations.

**Comment by snezhkaoliferenko** [comment]
in principle, it looks a bit like 'a' toll-like receptor.. i mean, without a bend - but a similar sort of arrangement of beta strands.
maybe it's not totally bonkers

**Comment by kimrutherford** [comment]
There's a bug in the range searches: the search button disappears if you type a max number then delete it even if there's a "min".

**Comment by ValWood** [comment]
> in principle, it looks a bit like 'a' toll-like receptor..
Yes, not surprising that an ER-PM contact site protein resembles a receptor that recognizes big ligands!

**Comment by kimrutherford** [comment]
It's still a work in progress but I've made an attempt at improving the range query interface.  I'd like help with the wording of the labels.

I haven't worked on the genome location query.  I plan to do that next.

It's available here if you'd like to try it: https://desktop.kmr.nz/query

![query-builder-improved-ranges-1](https://user-images.githubusercontent.com/90474/135224604-d1904173-cc27-47b1-abb8-2c135af7f4d9.png)

**Comment by ValWood** [comment]
Very nice the arrows indicating  more queries work well. 
You can  get  use feedback fromJuan/Jack tomorrow and we can tweak the wording.

**Comment by ValWood** [comment]
Proteins with any TM domain
Proteins with no TM domains

Protein feature 
Retrieve blah (for example short motifs (i.e. KEN, NLS) and signal peptides)

**Comment by mah11** [comment]
I'll edit the text and put it in a new ticket.

**Comment by kimrutherford** [comment]
These changes are in the main site now and I've just committed the configuration changes for japonicus, which will be visible in the morning.

I'll close this on Monday unless you spot any problems.

**Comment by snezhkaoliferenko** [comment]
Thank you!

**Comment by kimrutherford** [comment]
I'll close this and we can open new issues for problems.
